libs/ui/molecules/src/lib/AuctionLeftHearts/AuctionLeftHearts.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Icon, IconType, Row, Text } from '@inheartive/ui/atoms';
+
+interface IAuctionHeartsProps {
+  quantity: number;
+  authorName?: string;
+}
+
+function AuctionLeftHearts(props: IAuctionHeartsProps) {
+  const { quantity, authorName } = props;
+
+  return (
+    <Row space={1.5} alignItems='center'>
+      <Icon name={IconType.favoriteOutline} size={40} />
+      <Text>
+        {quantity} {authorName}
```

Comment:
Maybe it worth using default value for `authorName` since it might be `undefined` ?

Reply:
AutoherName it is the first name from the User, so I assumed this should be a required value in the database and shouldn't by undefined

Reply:
@BeataKula no problem, PR is still approved, `authorName` is optional property, this is why I thought that it might be `undefined`

Reply:
What is the purpose of this author name duplication here? I thought that maybe it's a name of the current winner of the auction and a price that he is paying for now. That makes more sense i think.

Reply:
@zgr92  I agree, but it is still first name of the user. I want to  add to auctionsMock more information, but in next task "Create Auction Template", so this duplication is temporary